metadata-ingestion/cli-ingestion.md

```diff
@@ -0,0 +1,94 @@
+# CLI Ingestion
+
+## Ingesting Metadata
+
+```shell
+pip install 'acryl-datahub[datahub-rest]'  # install the required plugin
+datahub ingest -c <path_to_recipe_file>
+```
+
+For more information on configuring recipes, please refer to [Recipes](recipe_overview.md)
```

Comment:
Is this intended to be reference material or a tutorial? Feels like we're mixing concerns here

Imo this should be a tutorial. The `--preview`, `--dry-run`, deploy, etc docs should get moved to the cli reference page (https://docs-website-ie0wod5ko-acryldata.vercel.app/docs/next/cli)

I think it's ok to show an example recipe here to lightly introduce the ideas of recipes/sources/sinks, and then link out to another page. Users should come away from this understanding all the core concepts at a high level (cli, recipes, sources, sinks) and should be ready to dive into source-specific docs (that should be the call to action at the end of the page)

The deploy command is actually more relevant for UI ingestion, since it lets users provision ingestion in the UI using the CLI